src/components/CategoriesView.jsx

```diff
@@ -0,0 +1,29 @@
+import "./CategoriesView.css";
+import { Link } from "react-router-dom";
```

Comment:
This import is unused. No need for it to be here. VSCode should show it in a lightened color to indicat that it is unused.